server/hotrod/src/main/java/org/infinispan/server/hotrod/HotRodServer.java

```diff
@@ -227,7 +227,7 @@ protected void startInternal(HotRodServerConfiguration configuration, EmbeddedCa
       queryFacade = queryFacades.size() > 0 ? queryFacades.get(0) : null;
       clientListenerRegistry = new ClientListenerRegistry(gcr.getComponent(EncoderRegistry.class),
             gcr.getComponent(ExecutorService.class, ASYNC_OPERATIONS_EXECUTOR));
-      clientCounterNotificationManager = new ClientCounterManagerNotificationManager(asCounterManager(cacheManager));
+      //clientCounterNotificationManager = new ClientCounterManagerNotificationManager(asCounterManager(cacheManager));
```

Comment:
Leftover?

Reply:
Ah, yes. Part of my temporarily aborted experiment to lazily start the CacheManager.